go.mod

```diff
@@ -1,5 +1,43 @@
 module tg-bridge
 
-go 1.22.6
+go 1.23.3
```

Comment:
the last go version is 1.24.5
but it could depend of way you will build the application.
for, instance if you use [buildpacks](https://buildpacks.io/) there could not be the last version of go supported. But let's for now keep the last stable version and then in some case will modify it for needs.